src/main/java/org/opensrp/web/rest/PlanResource.java

```diff
@@ -80,6 +79,18 @@
 
 	public static final String IS_TEMPLATE = "is_template";
 
+	public static final String PAGE_NUMBER = "pageNumber";
+
+	public static final String PAGE_SIZE = "pageSize";
+
+	public static final String PLAN_STATUS = "planStatus";
+
+	public static final String USE_CONTEXT = "useContext";
+
+	public static final String ORDER_BY_TYPE = "orderByType";
+
+	public static final String ORDER_BY_FIELD_NAME = "orderByFieldName";
```

Comment:
Define these in `org.opensrp.web.Constants` except `PLAN_STATUS` and `USE_CONTEXT` which are used on this resource only

Reply:
Done